docs/data/data-grid/state/RowSelectorPlayground.tsx

```diff
@@ -0,0 +1,550 @@
+            aria-label="basic tabs example"
+          >
+            <Tab label="Grid Example" />
+            <Tab label="Ids selector" />
+            <Tab label="Entity selector" />
+          </Tabs>
+        </Box>
+        <TabPanel value={tabPanemIndex} index={0}>
+          <Box sx={{ display: 'flex' }}>
+            <Box sx={{ width: '70%' }}>
+              <Box sx={{ width: '100%', display: 'flex', height: '3rem' }}>
+                <Typography>Maximum price</Typography>
+                <Slider
+                  sx={{ maxWidth: 200, ml: 2 }}
+                  size="small"
+                  defaultValue={900}
+                  aria-label="maximum price"
+                  valueLabelDisplay="auto"
+                  min={100}
+                  max={1200}
+                  onChangeCommitted={handleFilterChange}
+                  onChange={debouncedHandleFilterChange}
+                />
+              </Box>
+              <Box sx={{ height: '300px' }}>
+                <DataGridPro
+                  apiRef={apiRef}
+                  columns={columns}
+                  rows={rows}
+                  groupingColDef={{ headerName: 'Bicycle' }}
+                  treeData={hasTreeStructure}
+                  getTreeDataPath={getTreeDataPath}
+                  pageSize={hasPagination ? 2 : 100}
+                  rowsPerPageOptions={[2]}
+                  pagination={hasPagination}
+                  onSortModelChange={callRowSelector}
+                  onPageChange={callRowSelector}
+                  filterModel={filterModel}
+                  disableColumnMenu
+                />
+              </Box>
+            </Box>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                ml: 2,
+                maxWidth: '28%',
+              }}
+            >
+              <p style={{ height: '2rem', marginTop: '1rem', marginBottom: 0 }}>
+                Retruned ids
+              </p>
+              <List dense sx={{ overflow: 'auto', maxHeight: '300px' }}>
+                {selectedRows.map((id) => (
+                  <ListItem key={id}>
+                    <ListItemText primary={id} />
+                  </ListItem>
+                ))}
+              </List>
+            </Box>
+          </Box>
+        </TabPanel>
+        <TabPanel value={tabPanemIndex} index={1}>
+          <Box
+            sx={{
+              pl: 3,
+              backgroundColor: 'black',
+              color: 'white',
+              py: 1,
+            }}
+          >
+            <pre>
+              {`// Rows ids selector: 
+
+${formatedSelector(idsSelector)}`}
+            </pre>
+          </Box>
+        </TabPanel>
+        <TabPanel value={tabPanemIndex} index={2}>
+          <Box
+            sx={{
+              pl: 3,
+              backgroundColor: 'black',
+              color: 'white',
+              py: 1,
+            }}
+          >
+            <pre>
```

Comment:
You have an `HighlightedCode` component in `docs/src/modules/components/HighlightedCode` that could help you